ops/terraform/services/base/nonsensitive.tf

```diff
@@ -1,28 +1,16 @@
 locals {
   # High precedence. Establish override behavior by removing overrides if they're not provided, i.e. they're `null`
   common_nonsensitive_override_raw = {
-    "/bfd/${local.env}/common/nonsensitive/rds_snapshot_identifier"    = local.is_ephemeral_env ? data.aws_db_cluster_snapshot.seed[0].id : null
-    "/bfd/${local.env}/common/nonsensitive/ephemeral_environment_seed" = local.is_ephemeral_env ? var.ephemeral_environment_seed : null
+    "/bfd/${local.env}/common/nonsensitive/rds_snapshot_identifier" = local.is_ephemeral_env ? data.aws_db_cluster_snapshot.seed[0].id : null
   }
 
   # NOTE: null values are illegal, so we must strip them out if they should exist
   common_nonsensitive_override = { for key, value in local.common_nonsensitive_override_raw : key => value if value != null }
 
-  kms_key_alias = local.is_ephemeral_env ? "alias/bfd-${local.seed_env}-cmk" : "alias/bfd-${local.env}-cmk"
-
-  # NOTE: When instantiating an ephemeral environment, `local.common_nonsensitive_ssm` will be empty.
-  #       The var.epehemeral_environment_seed **must** be provided on the first run of terraform!
-  seed_env = lookup(
-    local.common_nonsensitive_ssm,
-    "/bfd/${local.env}/common/nonsensitive/ephemeral_environment_seed",
-    var.ephemeral_environment_seed
-  )
-
   # Normal precedence. Values stored in YAML files.
-  yaml_file = contains(local.established_envs, local.env) ? "${local.env}.yaml" : "ephemeral.yaml"
+  yaml_file = local.is_ephemeral_env ? "ephemeral.yaml" : "${local.env}.yaml"
   yaml = yamldecode(templatefile("${path.module}/values/${local.yaml_file}", {
-    env      = local.env
-    seed_env = local.seed_env
```

Comment:
`seed_env` is no longer required in the yaml configuration templates for SSM values.